apps/crn-frontend/src/network/users/UserProfile.tsx

```diff
@@ -50,6 +51,21 @@ const User: FC<Record<string, never>> = () => {
 
   const patchUserAvatar = usePatchUserAvatarById(userId);
   const [avatarSaving, setAvatarSaving] = useState(false);
+  const [sharedOutputsCount, setSharedOutputsCount] =
+    useState<number | undefined>();
+
+  const result = useResearchOutputs({
+    currentPage: 0,
+    filters: new Set(),
+    pageSize: 1,
+    searchQuery: '',
+    userId,
+  });
+
+  useEffect(() => {
+    setSharedOutputsCount(result.total || undefined);
+  }, [result.total]);
+
```

Comment:
I don't think we need to use State here (and no need for the useEffect).

i.e. 

const sharedOutputsCount = result.total || undefined;

or just inline it